js/main.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
```

Comment:
`Modify the counter so that it never resets: each time the user visits the page, the initial number of seconds he sees is the number of seconds he has already spent on the page. `